lib/MusicBrainz/Server/Edit/Release/Edit.pm

```diff
@@ -340,7 +340,7 @@ around allow_auto_edit => sub {
     return 0 if defined $self->data->{old}{language_id};
     return 0 if defined $self->data->{old}{script_id};
 
-    return 0 if defined $self->data->{old}{events};
+    return 0 if defined $self->data->{old}{events} && scalar $self->data->{old}{events} > 0;
```

Comment:
That should be `scalar @{ $self->data->{old}{events} }` if events is an array ref.

Reply:
Oops, fixed in https://github.com/metabrainz/musicbrainz-server/pull/561/commits/45fe51bbecf78561ba5832318ee61464be95f403